src/libs/antares/study/parts/common/cluster_list.hxx

```diff
@@ -0,0 +1,636 @@
+#ifndef __CLUSTER_LIST_COMMON_HXX__
```

Comment:
Not sure this file is useful, it seems never to be included (even in CMake files). Did I miss something ?
And the methods it defines are already defined in **cluster_list.cpp**

Reply:
removed